### PR TITLE
Transfer/77/receipt component

### PR DESCRIPTION
**Pull request by Zain-Ashfaq**

### CODE REVIEW

File: src/pages/Transfer/Transfer.jsx [hunk 1]
```diff
@@ -1,18 +1,65 @@
 import "./Transfer.scss";
+import Button from "../../components/Button/Button";
+const Transfer = ({
```
Comment (kirsty-nology) [inline src/pages/Transfer/Transfer.jsx:3]: add new line between imports and declaring component

File: src/pages/Transfer/Transfer.jsx [hunk 2]
```diff
@@ -1,18 +1,65 @@
...
+        <h1 className="transfer-info__transfer"> Transfer </h1>
+        <h1 className="transfer-info__heading"> Make Transfer </h1>
+        <p className="transfer-info__content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim nam
+          inventore deserunt id? Rem minima dolorem asperiores perferendis sed
+          aspernatur. Dolorem nostrum voluptatibus dolore odit provident minima
+          reiciendis dignissimos iste.
+        </p>
+      </div>
+      <div className="receipt">
```
Comment (kirsty-nology) [inline src/pages/Transfer/Transfer.jsx:23]: I would be tempted to extract the `receipt` section into a separate component

File: src/pages/Transfer/Transfer.scss [hunk 3]
```diff
@@ -1,34 +1,129 @@
...
+    .receipt__body__info--value {
+      font-size: 0.7rem;
+      width: auto;
+      color: $color-text;
+
+      font-family: $font-heading;
+      font-weight: 400;
+    }
+    #delivery {
+      //   inline-size: 8rem;
```
Comment (kirsty-nology) [inline src/pages/Transfer/Transfer.scss:84]: remove comment

File: src/pages/Transfer/Transfer.scss [hunk 4]
```diff
@@ -1,34 +1,129 @@
...
+    height: auto;
+    width: auto;
 
-    .transfer__amount {
-        grid-area: "amount";
+    .receipt__body__info--value {
+      font-size: 0.7rem;
+      width: auto;
+      color: $color-text;
+
```
Comment (kirsty-nology) [inline src/pages/Transfer/Transfer.scss:79]: remove extra line

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

File: src/pages/Transfer/Transfer.jsx [hunk 5]
```diff
@@ -1,18 +1,65 @@
 import "./Transfer.scss";
+import Button from "../../components/Button/Button";
+const Transfer = ({
+  sendCurr = "GBP - British Pounds",
+  sendAmount = "£1000.00",
```
Comment (LewisLovett) [inline src/pages/Transfer/Transfer.jsx:5]: sendAmount, recAmount and fee sould be just numbers. The money symbol (£,$) should be chosen based on sendCurr and recCurr

File: src/pages/Transfer/Transfer.jsx [hunk 6]
```diff
@@ -1,18 +1,65 @@
...
+            <h1 className="receipt__body__info--title">Delivery</h1>
+            <h1 className="receipt__body__info--value" id="delivery">
+              Typically same day
+            </h1>
+          </div>
+        </div>
+        <div className="receipt__bottom">
+          <div className="receipt__bottom__box">
+            <h1 className="receipt__bottom__box--info">Total</h1>
+            <h1 className="receipt__bottom__box--end">{sendAmount}</h1>
```
Comment (LewisLovett) [inline src/pages/Transfer/Transfer.jsx:54]: It think 'Total' should be sendAmount + fee

**Review by LewisLovett — CHANGES_REQUESTED** [review]

File: src/pages/Transfer/Transfer.jsx (hunk 1 above)
Comment (Zain-Ashfaq) [reply src/pages/Transfer/Transfer.jsx:3]: done

File: src/pages/Transfer/Transfer.scss (hunk 3 above)
Comment (Zain-Ashfaq) [reply src/pages/Transfer/Transfer.scss:84]: done

File: src/pages/Transfer/Transfer.scss (hunk 4 above)
Comment (Zain-Ashfaq) [reply src/pages/Transfer/Transfer.scss:79]: done

File: src/pages/Transfer/Transfer.jsx (hunk 6 above)
Comment (Zain-Ashfaq) [reply src/pages/Transfer/Transfer.jsx:54]: Done

**Review by kirsty-nology — APPROVED** [review]

**Review by LewisLovett — APPROVED** [review]

**Pull request merged**